src/FieldArrayProps.types.js.flow

```diff
@@ -1,9 +1,9 @@
 // @flow
-import * as React from 'react'
+import type {ElementType, ComponentType} from 'react'
 
 export type Props = {
   name: string,
-  component: React.ComponentType<*> | React.ElementType,
+  component: ComponentType<*> | tType,
```

Comment:
This is breaking Travis, see https://travis-ci.org/erikras/redux-form/jobs/407284062#L6161